### ISSUE TITLE
klt killercoda demo not accessible 

### ISSUE THREAD

**Issue by SuperAayush** [keptn/killercoda#1]
Under the keptn org on killercoda website I see the end to end delivery tutorial but unable to see the klt demo.
![klt](https://user-images.githubusercontent.com/78820926/235914222-0ccfca5e-4673-4286-bc7c-201194b79ea8.png)

Is it like something to be done from the killercoda end?

cc - @agardnerIT

**Comment by agardnerIT** [comment]
Yes. Currently the demo and `https://killercoda.com/keptn` is still pointing to `https://github.com/afzal442/keptn-pg-killerway`

I've opened a ticket with the Killercoda team on how best to migrate towards the official repo since Afzal has already "reserved" the `/keptn` username - so we'll need some sort of synchronised move.

I'm on it :)

**Comment by agardnerIT** [comment]
@staceypotter Do you have admin rights to this repo? If so, can you configure the following please?

1) Set up the GitHub webhook so that the sync to Killercoda works ([as described here](https://killercoda.com/creators/get-started))
2) Set the username to `keptn-new` (NOT `keptn` yet!) in the killercoda account tied to this repo.

Doing these two steps will make the tutorials available on `https://killercoda.com/keptn-new` whereby we can test things and ensure it's all working smoothly.

------
DO NOT ACTION ANYTHING BELOW THIS LINE (YET) !!

Once that's done, we will need to co-ordinate with @afzal442 to:

1) Afzal removes the `keptn` username from his killercoda account (thus it becomes "available" again)
2) We change `keptn-new` to `keptn`

If we do it right, the switchover should be seamless and this will be the official Keptn killercoda repo from that point.

**Comment by DavidPHirsch** [comment]
@agardnerIT @afzal442 we need the secret to be able to add the webhook

**Comment by DavidPHirsch** [comment]
@agardnerIT @SuperAayush should be taken care of now, new account is https://killercoda.com/keptnproject
